src/guest/vm_builder_qemu.cc

```diff
@@ -222,12 +229,20 @@ static int SetAvailableVf(void) {
         return false;
     }
 
-    for (int i = 1; i < totalvfs; i++) {
+    for (int i = 1; i < current_vfs; i++) {
         std::string sd(kIntelGpuDevPath);
         sd.replace(sd.end() - 2, sd.end(), std::to_string(i) + "/enable");
         int status = ReadSysFile(sd.c_str(), std::ios_base::dec);
-        if (status == 0)
+        if (status == 0) {
+            std::string timeoutPath = kTimeoutPath;
```

Comment:
One more question: does the timeout/quantum depend on specific kernel version?

Reply:
No. We have been using it on different kernels till now for multiple releases till now and not faced any issue.